backend/src/db/migrate/store.js

```diff
@@ -1,11 +1,62 @@
 import { getDriver, getNeode } from '../../db/neo4j'
+import { hashSync } from 'bcryptjs'
+import { v4 as uuid } from 'uuid'
+
+const defaultAdmin = {
+  email: 'admin@example.org',
+  password: hashSync('1234', 10),
+  name: 'admin',
+  id: uuid(),
+  slug: 'admin',
+}
+
+const createDefaultAdminUser = async (session) => {
+  const readTxResultPromise = session.readTransaction(async (txc) => {
+    const result = await txc.run(
+      'MATCH (email:EmailAddress)-[:BELONGS_TO]->(user:User) RETURN count(user) AS userCount',
+    )
+    return result.records.map((r) => r.get('userCount'))
+  })
+  let createAdmin = false
+  try {
+    const userCount = parseInt(String(await readTxResultPromise))
+    if (userCount === 0) createAdmin = true
+  } catch (error) {
+    console.log(error) // eslint-disable-line no-console
+  }
+  if (createAdmin) {
+    const createAdminTxResultPromise = session.writeTransaction(async (txc) => {
+      txc.run(
+        `MERGE (e:EmailAddress {
+           email: "${defaultAdmin.email}",
+           createdAt: datetime()
+         })-[:BELONGS_TO]->(u:User {
+           name: "${defaultAdmin.name}",
+           encryptedPassword: "${defaultAdmin.password}",
+           role: "admin",
+           id: "${defaultAdmin.id}",
+           slug: "${defaultAdmin.slug}",
+           createdAt: toString(datetime()),
+           deleted: false,
+           disabled: false
+         })-[:PRIMARY_EMAIL]->(e)`,
```

Comment:
idealy we would make a file out of all queries instead of putting them in the code - this would have the potential benefit of reuse. Ignore this for now